### PR TITLE
Adding press release for v6

### PR DESCRIPTION
**Pull request by mikeal**

### CODE REVIEW

**Comment by jasnell** [comment]
LGTM. Do you want this to post before or after the release post?

**Comment by mikeal** [comment]
I'm gonna pull the trigger now because there are articles already referencing it because the embargo was lifted at 8am :/

File: locale/en/blog/announcements/v6-release.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+version numbers of software libraries similar to dependencies. Under the Node.js’ [Long-Term 
+Support (LTS)](https://github.com/nodejs/LTS/), version 6 is now the “Current” release line 
+while version 5 will be maintained for a few more months. In October 2016, Node.js v6 will 
+become the LTS release and the LTS release line (version 4) will go under maintenance mode in 
+April 2017, meaning only critical bugs, critical security fixes and documentation updates 
+will be permitted. Users should begin transitioning from v4 to v6 in October when v6 goes 
+into LTS.
+
+Additional Resources
+* [Download version 6](https://nodejs.org/download/release/v6.0.0/)
```
Comment (MylesBorins) [inline locale/en/blog/announcements/v6-release.md:65]: could this be replaced with a coming soon link until the download is ready?

**Comment by jasnell** [comment]
+1... release is almost ready to go

**Comment by MylesBorins** [comment]
I added a small nit, but perhaps too late. Should be replace the download link, which currenty doesn't work, with coming soon?

**Comment by ghost** [comment]
@TheAlphaNerd absolutely, yeah

**Comment by ghost** [comment]
#690 is up for this

**Pull request merged**